### DISCUSSION TITLE
Group tablist by permission rank

### DISCUSSION THREAD

**Discussion by Bloodlex** [Zrips/CMI#6210] [Feature Requests]
Would it be possible to add sorting to tablist so Admins are on top, Mods beneath and so on? Thank you 🙂

**Comment by CloudeLecaw** [comment]
Long wished 1000 times,.and 1000 times was adding in discussion..so this.time too.;o

Would be more effictiv if users would rate and comment the discussion instead of opening a new one each time. So Open the Discussion tab, type in TAB and you will find enough other once where u might can rate and comment  this to upvote it.

**Comment by mrfloris** [comment]
https://github.com/Zrips/CMI/discussions/931

**Comment by mrfloris** [comment]
Tablist.yml has player-group sorting in the latest builds of cmilib and cmi. Upgrade and review tablist.yml's configuration (bottom of page). You can sort ascending or descending by type, and types range from permission node weight to baltop rank, etc.
